framework/source/class/qx/ui/form/AbstractField.js

```diff
@@ -935,31 +935,33 @@ qx.Class.define("qx.ui.form.AbstractField",
         qx.ui.form.AbstractField.__addPlaceholderRules();
       }
     },
+
     /**
-     * validates the the input value
+     * Validates the the input value.
      * 
-     * @param {type} value: the value to check
-     * @returns the checked value
+     * @param value {Object} The value to check
```

Comment:
`{Object}` ? Really? This is like saying "anything". I am pretty sure this needs to be `{String|null}`